outline.md

```diff
@@ -52,15 +56,23 @@ Now that you have an Ember-CLI generated project, let's step inside and start ex
     README.md               bower_components        package.json            tests
     app                     config                  public                  vendor
 
-##### 1. Package.json and node_modules
-The first things to notice is the file package.json and the directory node_modules.  These are from npm, and if you're new to NPM, take a look at what is in the package.json file.  This file contains information about packaging up your application as a module itself, but more importantly for our purposes it contains information about what npm modules are required to run and develop our app.  As we're building a client-side application, we're unlikely to require any npm modules to run it, but all of the various ember/brocolli/etc tools that we end up needing will be specified in this package.json and installed into the node_modules directory.
+##### 1. package.json and node_modules
+The first things to notice is the file `package.json` and the directory `node_modules`. These are from npm, and if you're new to NPM, take a look at what is in the `package.json` file.  This file contains information about packaging up your application as a module itself, but more importantly for our purposes it contains information about what npm modules are required to run and develop our app. When using ember-cli you won't often come in here and edit this ourselves directly however you'll see that the packages needed for broccoli and ember-cli are specified here. If you were to install any ember-cli-addons yourself, you would see them show up in here as well. The packages specified in `package.json` will be installed under `node_modules/`.
+
 ##### 2. bower.json and bower_components
-The next thing to look at is the file bower.json and the bower_components directory.  Bower has become the defacto standard in package management for front end applications, and our Ember-CLI application will use it to manage our dependencies.  If you open up that file you'll see that our application comes out of the box with not only Ember itself but jQuery, Ember Data (a powerful data persistence library), and qunit.
+The next thing to look at is the file bower.json and the bower_components directory. This coupling is similar to that of the prior. Bower has become the defacto standard in package management for front end applications and our Ember-CLI application will use it to manage our dependencies. If you open up that file you'll see that our application comes out of the box with not only Ember itself but jQuery, Ember Data (a powerful data persistence library), and qunit (testing framework).
```

Comment:
Let's be consistent... I like the new emphasis in section 1 of `package.json` instead of just package.json... so let's add that for bower.json and bower_components